### PR TITLE
remove sub command

### PR DESCRIPTION
**Pull request by saeideng**

### CODE REVIEW

File: src/Console/ConsoleOptionParser.php [hunk 1]
```diff
@@ -772,65 +628,25 @@ public function parse(array $argv, ?ConsoleIo $io = null): array
...
-            if ($format === 'xml') {
-                return (string)$formatter->xml();
-            }
+        if ($format === 'text') {
+            return $formatter->text($width);
+        }
+        if ($format === 'xml') {
+            return (string)$formatter->xml();
         }
-        $subcommand = (string)$subcommand;
```
Comment (markstory) [inline src/Console/ConsoleOptionParser.php:797]: We need a return or a throw here to make respect the return types.

Comment (saeideng) [reply src/Console/ConsoleOptionParser.php:797]: Done.

Comment (saeideng) [reply src/Console/ConsoleOptionParser.php:797]: I also removed the `subcommand` param

**Comment by saeideng** [comment]
Is documentation of changes required?
@markstory

**Pull request merged**